src/main/java/com/vi/appointmentservice/repository/CalcomRepository.java

```diff
@@ -79,4 +79,11 @@ public void deleteAttendeeWithoutBooking() {
     calcomDBNamedParamterTemplate.update(QUERY, new MapSqlParameterSource());
   }
 
+  public void updateAttendeeEmail(final Long bookingId, final String email) {
```

Comment:
would probably be more performant in case of "where in".

Reply:
done